data/graphics.js

```diff
@@ -9656,9 +9642,8 @@ var BattleMoveAnims = {
 			});
 			defender.delay(200);
 			defender.anim({
-				x: defender.leftof(+5),
```

Comment:
Here's your problem. You're using `leftof`. Why? In theory it should only be forwards and backwards movement... what's the intent of the effect?